services/cedar.go

```diff
@@ -56,24 +58,27 @@ func DialCedar(ctx context.Context, client *http.Client, opts *DialCedarOptions)
 
 	httpAddress := "https://" + opts.BaseAddress
 
+	apiHeaders := map[string]string{}
+	if opts.Username != "" && opts.APIKey != "" {
+		apiHeaders["Api-User"] = opts.Username
```

Comment:
Should `Api-User` and `Api-Key` be constants somewhere?

Reply:
Changed them to be constants.